zerver/lib/actions.py

```diff
@@ -1439,6 +1435,15 @@ def internal_send_message(realm, sender_email, recipient_type_name, recipients,
 
     do_send_messages([msg])
 
+def internal_send_private_message(realm, sender, recipient_email, content):
+    # type: (Realm, UserProfile, Text, Text) -> None
+    msg = internal_prep_private_message(realm, sender, recipient_email, content)
```

Comment:
I think we prefer `message` to `msg`. Also, I'd remove all the extra newlines. 

ah, I see you copied it from `internal_send_message` above, but I guess styles have changed since then :).